build/helm/dc_website_v1/values.yaml

```diff
@@ -0,0 +1,87 @@
+# Default values for dc-website.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Website service config.
+website:
+  # The values of the doamins that should point to the DC website instance.
+  # The DNS should be configured to have an "A" record that points to
+  # the IP attached in the ingress section.
+  domains:
```

Comment:
is "domains" intended to be plural here?

Reply:
Yes, intentional because both k8s and gke supports multiple domains and I thought there would be no point limiting just the user input. On the other hand, likely 99% of the time the domains will be a list of 1 item, which could be confusing. Thoughts?

Reply:
Can start with single "domain" here and change it to domains when the need comes?